test/e2e/base/manifests.yaml

```diff
@@ -47,7 +47,6 @@ spec:
       group: gateway.envoyproxy.io
       kind: EnvoyProxy
       name: zipkin-tracing
-      namespace: envoy-gateway-system
```

Comment:
rm namespace cause this field is no longer supported https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.LocalParametersReference